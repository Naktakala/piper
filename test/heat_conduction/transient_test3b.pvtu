<VTKFile type="PUnstructuredGrid" version="0.1" byte_order="LittleEndian" header_type="UInt32" compressor="vtkZLibDataCompressor">
  <PUnstructuredGrid GhostLevel="0">
    <PPointData>
      <PDataArray type="Float64" Name="T"/>
    </PPointData>
    <PCellData>
      <PDataArray type="Int32" Name="Material"/>
      <PDataArray type="UInt32" Name="Partition"/>
      <PDataArray type="Float64" Name="T"/>
    </PCellData>
    <PPoints>
      <PDataArray type="Float64" Name="Points" NumberOfComponents="3"/>
    </PPoints>
    <Piece Source="transient_test3b_0.vtu"/>
    <Piece Source="transient_test3b_1.vtu"/>
    <Piece Source="transient_test3b_2.vtu"/>
    <Piece Source="transient_test3b_3.vtu"/>
    <Piece Source="transient_test3b_4.vtu"/>
    <Piece Source="transient_test3b_5.vtu"/>
    <Piece Source="transient_test3b_6.vtu"/>
    <Piece Source="transient_test3b_7.vtu"/>
  </PUnstructuredGrid>
</VTKFile>
