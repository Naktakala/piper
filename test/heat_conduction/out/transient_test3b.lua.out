mpiexec -np 8 /Users/vermjc/Repos/ChiTech/piper/bin/piper transient_test3b.lua --suppress_color --supress_beg_end_timelog master_export=false 
[0]  Parsing argument 1 transient_test3b.lua
[0m[0]  Parsing argument 2 --suppress_color
[0m[0]  Parsing argument 3 --supress_beg_end_timelog
[0]  Parsing argument 4 master_export=false
[0]  ChiTech number of arguments supplied: 4
[0]  Computing cell-centroids.
[0]  Done computing cell-centroids.
[0]  Checking cell-center-to-face orientations
[0]  Done checking cell-center-to-face orientations
[0]  00:00:00 Establishing cell connectivity.
[0]  00:00:00 Vertex cell subscriptions complete.
[0]  00:00:00 Surpassing cell 1000 of 10000 (10%)
[0]  00:00:00 Surpassing cell 2000 of 10000 (20%)
[0]  00:00:00 Surpassing cell 3001 of 10000 (30%)
[0]  00:00:00 Surpassing cell 4000 of 10000 (40%)
[0]  00:00:00 Surpassing cell 5000 of 10000 (50%)
[0]  00:00:00 Surpassing cell 6001 of 10000 (60%)
[0]  00:00:00 Surpassing cell 7001 of 10000 (70%)
[0]  00:00:00 Surpassing cell 8000 of 10000 (80%)
[0]  00:00:00 Surpassing cell 9000 of 10000 (90%)
[0]  00:00:00 Surpassing cell 10000 of 10000 (100%)
[0]  00:00:00 Establishing cell boundary connectivity.
[0]  00:00:00 Done establishing cell connectivity.
[0]  MeshGenerator: Cells created = 10000
[0]  Time step 0, time = 0, dt = 0.001
[0]  Setting initial solution
[0]  Solver "phys1" Time step 0, time = 0.001, dt = 0.001
[0]  Solver "phys1" Time step 1, time = 0.002, dt = 0.001
[0]  Solver "phys1" Time step 2, time = 0.003, dt = 0.001
[0]  Solver "phys1" Time step 3, time = 0.004, dt = 0.001
[0]  Solver "phys1" Time step 4, time = 0.005, dt = 0.001
[0]  Solver "phys1" Time step 5, time = 0.006, dt = 0.001
[0]  Solver "phys1" Time step 6, time = 0.007, dt = 0.001
[0]  Solver "phys1" Time step 7, time = 0.008, dt = 0.001
[0]  Solver "phys1" Time step 8, time = 0.009, dt = 0.001
[0]  Solver "phys1" Time step 9, time = 0.01, dt = 0.001
[0]  Solver "phys1" Time step 10, time = 0.011, dt = 0.001
[0]  Solver "phys1" Time step 11, time = 0.012, dt = 0.001
[0]  Solver "phys1" Time step 12, time = 0.013, dt = 0.001
[0]  Solver "phys1" Time step 13, time = 0.014, dt = 0.001
[0]  Solver "phys1" Time step 14, time = 0.015, dt = 0.001
[0]  Solver "phys1" Time step 15, time = 0.016, dt = 0.001
[0]  Solver "phys1" Time step 16, time = 0.017, dt = 0.001
[0]  Solver "phys1" Time step 17, time = 0.018, dt = 0.001
[0]  Solver "phys1" Time step 18, time = 0.019, dt = 0.001
[0]  Solver "phys1" Time step 19, time = 0.02, dt = 0.001
[0]  Solver "phys1" Time step 20, time = 0.021, dt = 0.001
[0]  Solver "phys1" Time step 21, time = 0.022, dt = 0.001
[0]  Solver "phys1" Time step 22, time = 0.023, dt = 0.001
[0]  Solver "phys1" Time step 23, time = 0.024, dt = 0.001
[0]  Solver "phys1" Time step 24, time = 0.025, dt = 0.001
[0]  Solver "phys1" Time step 25, time = 0.026, dt = 0.001
[0]  Solver "phys1" Time step 26, time = 0.027, dt = 0.001
[0]  Solver "phys1" Time step 27, time = 0.028, dt = 0.001
[0]  Solver "phys1" Time step 28, time = 0.029, dt = 0.001
[0]  Solver "phys1" Time step 29, time = 0.03, dt = 0.001
[0]  Solver "phys1" Time step 30, time = 0.031, dt = 0.001
[0]  Solver "phys1" Time step 31, time = 0.032, dt = 0.001
[0]  Solver "phys1" Time step 32, time = 0.033, dt = 0.001
[0]  Solver "phys1" Time step 33, time = 0.034, dt = 0.001
[0]  Solver "phys1" Time step 34, time = 0.035, dt = 0.001
[0]  Solver "phys1" Time step 35, time = 0.036, dt = 0.001
[0]  Solver "phys1" Time step 36, time = 0.037, dt = 0.001
[0]  Solver "phys1" Time step 37, time = 0.038, dt = 0.001
[0]  Solver "phys1" Time step 38, time = 0.039, dt = 0.001
[0]  Solver "phys1" Time step 39, time = 0.04, dt = 0.001
[0]  Solver "phys1" Time step 40, time = 0.041, dt = 0.001
[0]  Solver "phys1" Time step 41, time = 0.042, dt = 0.001
[0]  Solver "phys1" Time step 42, time = 0.043, dt = 0.001
[0]  Solver "phys1" Time step 43, time = 0.044, dt = 0.001
[0]  Solver "phys1" Time step 44, time = 0.045, dt = 0.001
[0]  Solver "phys1" Time step 45, time = 0.046, dt = 0.001
[0]  Solver "phys1" Time step 46, time = 0.047, dt = 0.001
[0]  Solver "phys1" Time step 47, time = 0.048, dt = 0.001
[0]  Solver "phys1" Time step 48, time = 0.049, dt = 0.001
[0]  Solver "phys1" Time step 49, time = 0.05, dt = 0.001
[0]  Solver "phys1" Time step 50, time = 0.051, dt = 0.001
[0]  Solver "phys1" Time step 51, time = 0.052, dt = 0.001
[0]  Solver "phys1" Time step 52, time = 0.053, dt = 0.001
[0]  Solver "phys1" Time step 53, time = 0.054, dt = 0.001
[0]  Solver "phys1" Time step 54, time = 0.055, dt = 0.001
[0]  Solver "phys1" Time step 55, time = 0.056, dt = 0.001
[0]  Solver "phys1" Time step 56, time = 0.057, dt = 0.001
[0]  Solver "phys1" Time step 57, time = 0.058, dt = 0.001
[0]  Solver "phys1" Time step 58, time = 0.059, dt = 0.001
[0]  Solver "phys1" Time step 59, time = 0.06, dt = 0.001
[0]  Solver "phys1" Time step 60, time = 0.061, dt = 0.001
[0]  Solver "phys1" Time step 61, time = 0.062, dt = 0.001
[0]  Solver "phys1" Time step 62, time = 0.063, dt = 0.001
[0]  Solver "phys1" Time step 63, time = 0.064, dt = 0.001
[0]  Solver "phys1" Time step 64, time = 0.065, dt = 0.001
[0]  Solver "phys1" Time step 65, time = 0.066, dt = 0.001
[0]  Solver "phys1" Time step 66, time = 0.067, dt = 0.001
[0]  Solver "phys1" Time step 67, time = 0.068, dt = 0.001
[0]  Solver "phys1" Time step 68, time = 0.069, dt = 0.001
[0]  Solver "phys1" Time step 69, time = 0.07, dt = 0.001
[0]  Solver "phys1" Time step 70, time = 0.071, dt = 0.001
[0]  Solver "phys1" Time step 71, time = 0.072, dt = 0.001
[0]  Solver "phys1" Time step 72, time = 0.073, dt = 0.001
[0]  Solver "phys1" Time step 73, time = 0.074, dt = 0.001
[0]  Solver "phys1" Time step 74, time = 0.075, dt = 0.001
[0]  Solver "phys1" Time step 75, time = 0.076, dt = 0.001
[0]  Solver "phys1" Time step 76, time = 0.077, dt = 0.001
[0]  Solver "phys1" Time step 77, time = 0.078, dt = 0.001
[0]  Solver "phys1" Time step 78, time = 0.079, dt = 0.001
[0]  Solver "phys1" Time step 79, time = 0.08, dt = 0.001
[0]  Solver "phys1" Time step 80, time = 0.081, dt = 0.001
[0]  Solver "phys1" Time step 81, time = 0.082, dt = 0.001
[0]  Solver "phys1" Time step 82, time = 0.083, dt = 0.001
[0]  Solver "phys1" Time step 83, time = 0.084, dt = 0.001
[0]  Solver "phys1" Time step 84, time = 0.085, dt = 0.001
[0]  Solver "phys1" Time step 85, time = 0.086, dt = 0.001
[0]  Solver "phys1" Time step 86, time = 0.087, dt = 0.001
[0]  Solver "phys1" Time step 87, time = 0.088, dt = 0.001
[0]  Solver "phys1" Time step 88, time = 0.089, dt = 0.001
[0]  Solver "phys1" Time step 89, time = 0.09, dt = 0.001
[0]  Solver "phys1" Time step 90, time = 0.091, dt = 0.001
[0]  Solver "phys1" Time step 91, time = 0.092, dt = 0.001
[0]  Solver "phys1" Time step 92, time = 0.093, dt = 0.001
[0]  Solver "phys1" Time step 93, time = 0.094, dt = 0.001
[0]  Solver "phys1" Time step 94, time = 0.095, dt = 0.001
[0]  Solver "phys1" Time step 95, time = 0.096, dt = 0.001
[0]  Solver "phys1" Time step 96, time = 0.097, dt = 0.001
[0]  Solver "phys1" Time step 97, time = 0.098, dt = 0.001
[0]  Solver "phys1" Time step 98, time = 0.099, dt = 0.001
[0]  Solver "phys1" Time step 99, time = 0.1, dt = 0.001
[0]  
[0]  SCALAR post-processors latest values at event "ProgramExecuted"
[0]  *----------------------*-----------------*
[0]  | Post-Processor Name  | Value           |
[0]  *----------------------*-----------------*
[0]  | maxval(latest)       |      146.342584 |
[0]  *----------------------*-----------------*
[0]  


